app/test_resources.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+from main import app
+
+class RegistrationTest(unittest.TestCase):
+
+    def setUp(self):
+        self.app = app.test_client()
+
+    def test_should_success_registration(self):
+        payload = dict(
```

Comment:
Sometimes you use literal dict construction and sometimes you call constructor. To be consistent unless you have a good reason to use a function use dict literal (and there are really few good reasons as dict literals now support merging and dict expansion)